### PR TITLE
Add support for custom config files

### PR DESCRIPTION
**Pull request by EliotBerriot**
This is a simple PR that introduce a new `--config` flag in the CLI, which accept a path to a custom config file. 

We needed this in one project :)

### CODE REVIEW

File: src/towncrier/_settings.py [hunk 1]
```diff
@@ -22,7 +22,13 @@
 
 
 def load_config(from_dir):
-    fn = os.path.join(from_dir, "pyproject.toml")
+    if os.path.isfile(from_dir):
+        # maybe from_dir is actually a config_file path
+        fn = from_dir
+    else:
+        # we fallback to default behaviou of finding the config_file
```
Comment (ewjoachim) [inline src/towncrier/_settings.py]: behaviour

Comment (EliotBerriot) [reply src/towncrier/_settings.py]: Fixed thanks :)

**Comment by EliotBerriot** [comment]
We don't need or use this anymore, so feel free to close the PR :)

**Comment by hawkowl** [comment]
It's kind of intended to have it in the one place to match with the pyproject PEP, so I am not sure I'd want to change it. Thanks though, I'll reconsider if more people think it's an idea.

**Pull request closed without merging**